ansible/roles/operator-pipeline/templates/openshift/tasks/get-cert-project-related-data.yml

```diff
@@ -57,3 +60,5 @@ spec:
         cat cert_project | jq '.org_id' | tr -d '"' |  tr -d '"' | grep . | tr -d "\n" | tee $(results.org_id.path)
         cat cert_project | jq '.contacts[].email_address' | tr '\n' ' ' | tr -d '"' | grep . | tr -d "\n" | tee $(results.contacts.path)
         cat cert_project | jq '.operator_distribution' |  tr -d '"' | grep . | tr -d "\n" | tee $(results.operator_distribution.path)
+        echo -n "none" | tee $(results.certification_status.path)
+        echo -n "false" | tee $(results.update_cert_project_status.path)
```

Comment:
If the current certification_status is `Waiting for container image` (or a number of other possible allowed values), this won't correct it which isn't ideal. I think the logic may need to be changed to the following:

```
#pseudo code
if current_status != "Published":
    new_status = "In Progress"
    update_status = true
else:
   new_status = none
   update_status = false
```

Reply:
I was following this AC info https://issues.redhat.com/browse/ISV-871. In this task, it is mentioned that only if the state is `Started` then we need to update the state to `In Progress`.

Reply:
Unfortunately that's no longer sufficient because of the prevalence of other incorrect/invalid states in the data.